selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Menu.java

```diff
@@ -106,29 +108,33 @@ public Boolean apply(WebDriver driver) {
    * @param idCommandName
    */
   public void runCommand(final String idTopMenuCommand, final String idCommandName) {
+    WebDriverWait redrawMenuItemsWait =
```

Comment:
Let's extract field **redrawMenuItemsWait** to avoid repetitive creation of it in different methods in class.